plugins/chat/test/javascripts/components/chat-notices-test.js

```diff
@@ -0,0 +1,53 @@
+import ChatChannel from "discourse/plugins/chat/discourse/models/chat-channel";
+import { setupRenderingTest } from "discourse/tests/helpers/component-test";
+import hbs from "htmlbars-inline-precompile";
+import { query, queryAll } from "discourse/tests/helpers/qunit-helpers";
+import { module, test } from "qunit";
+import { click, render } from "@ember/test-helpers";
+
+module("Discourse Chat | Component | chat-notice", function (hooks) {
+  setupRenderingTest(hooks);
+
+  test("displays all notices for a channel", async function (assert) {
+    this.channel = ChatChannel.create({ id: 12, chatable_type: "Category" });
```

Comment:
you have `fabricators.channel()` which should work, if it doesn’t don't sweat it, I will look in a followup PR